search/search_tests/house_detector_tests.cpp

```diff
@@ -386,7 +387,7 @@ UNIT_TEST(HS_MWMSearch)
   }
 
   Index index;
-  auto p = index.Register(LocalCountryFile::MakeForTesting(country));
+  auto p = index.Register(LocalCountryFile::MakeForTesting(country, version::kASingleMwmVersionForTesting1));
```

Comment:
Why not to set a default value (kSingleMwmVersionForTesting1) for a second argument of MakeForTesting()? The size of this PR would be significantly smaller.